python/sdist/amici/sbml_import.py

```diff
@@ -1631,7 +1662,7 @@ def get_empty_bolus_value() -> sp.Float:
                         "Could not process event assignment for "
                         f"{str(variable_sym)}. AMICI currently only allows "
                         "event assignments to species; parameters; or, "
-                        "compartments with rate rules, at the moment."
+                        "compartments, at the moment."
```

Comment:
```suggestion
                        "compartments."
```
already have "currently" at the beginning of the sentence.